gcc/rust/typecheck/rust-hir-type-check-base.h

```diff
@@ -163,12 +162,12 @@ class TypeCheckBase : public HIR::HIRVisitor
   virtual void visit (HIR::MacroMatcher &) override {}
   virtual void visit (HIR::MacroRulesDefinition &) override {}
   virtual void visit (HIR::MacroInvocation &) override {}
-  virtual void visit (HIR::MetaItemPath &) override {}
-  virtual void visit (HIR::MetaItemSeq &) override {}
-  virtual void visit (HIR::MetaWord &) override {}
-  virtual void visit (HIR::MetaNameValueStr &) override {}
-  virtual void visit (HIR::MetaListPaths &) override {}
-  virtual void visit (HIR::MetaListNameValueStr &) override {}
+  // virtual void visit (HIR::MetaItemPath &) override {}
```

Comment:
Same question here ?

Reply:
I thought that's better for consistency to comment the visit like before. But yes, I can simply remove them all.

Reply:
Maybe there's a reason for these comments (@philberty ?) but I guess they are not really needed (the visitor pattern being widely used in gccrs, it's easy to find which methods can be overridden)